skyportal/utils/cache.py

```diff
@@ -0,0 +1,90 @@
+from pathlib import Path
+import hashlib
+import os
+
+from baselayer.log import make_log
+
+log = make_log('cache')
+
+
+class Cache:
+    def __init__(self, cache_dir, max_items):
```

Comment:
should `max_items` be a keyword (e.g. default to 10)?

Reply:
Updated.